code/modules/surgery/organs/internal/heart/_heart.dm

```diff
@@ -93,7 +93,7 @@
 	return beating
 
 /obj/item/organ/heart/get_status_text(advanced, add_tooltips)
-	if(!beating && !(organ_flags & ORGAN_FAILING) && owner.needs_heart() && owner.stat != DEAD)
+	if((!beating && !(organ_flags & ORGAN_FAILING) && owner.needs_heart() && owner.stat != DEAD) || owner.has_status_effect(/datum/status_effect/heart_attack))
 		return conditional_tooltip("<font color='#cc3333'>Cardiac Arrest</font>", "Apply defibrillation immediately. Similar electric shocks may work in emergencies.", add_tooltips)
```

Comment:
Talking about this in particular re: last comment

Reply:
![image](https://github.com/user-attachments/assets/0a88d7b3-b770-4b1e-8040-e3e75bb5196f)

Slightly different message for dead players added.